CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## master
 
+## [v5.2.0][v5.2.0] (December 10, 2021)
+
+* Refactored gin middleware([#224](https://github.com/airbrake/gobrake/pull/224))
+* Added the support for using severity levels defined by [Apex/log](https://github.com/apex/log) middleware instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))
```

Comment:
This sentence feels wrong to me. I'll let @scottsbaldwin check it because I am not a native English speaker.

Reply:
nor is he :)

Reply:
```suggestion
* Use [Apex/log](https://github.com/apex/log) severity levels instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))
```